### PR TITLE
Bump eventing components to 1.9

### PR DESCRIPTION
**Pull request by pierDipi**
As per title.

Istio bump in a follow up PR (https://github.com/openshift-knative/serverless-operator/pull/2069/)

### CODE REVIEW

File: knative-operator/deploy/resources/knativekafka/broker/eventing-kafka-broker.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ metadata:
   name: config-kafka-broker-data-plane
   namespace: knative-eventing
   labels:
-    kafka.eventing.knative.dev/release: v1.8
+    app.kubernetes.io/version: devel
```
Comment (pierDipi) [inline knative-operator/deploy/resources/knativekafka/broker/eventing-kafka-broker.yaml:21]: I'm working on the version fix

File: knative-operator/deploy/resources/knativekafka/source/eventing-kafka-source.yaml [hunk 2]
```diff
@@ -367,7 +370,7 @@ spec:
             - name: WAIT_STARTUP_SECONDS
               value: "8"
             - name: JAVA_TOOL_OPTIONS
-              value: "-XX:+CrashOnOutOfMemoryError"
+              value: "-XX:+CrashOnOutOfMemoryError -XX:InitialRAMPercentage=70.0 -XX:MinRAMPercentage=70.0 -XX:MaxRAMPercentage=70.0"
```
Comment (pierDipi) [inline knative-operator/deploy/resources/knativekafka/source/eventing-kafka-source.yaml:373]: This should be patched

**Review by creydr — APPROVED** [review]
/lgtm

**Pull request merged**